src/Controller/StudentInfoController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+
+class StudentInfoController extends Controller
+{
+    private $data_file = 'data.json';
+
+    /**
+     * @Route("/student/info", name="student_info")
+     */
+    public function index()
+    {
+        $data = file_get_contents($this->data_file);
+        $teams = json_decode($data);
+        $request = Request::createFromGlobals();
```

Comment:
Pabandyk `Request` perkelti į argumentus: https://symfony.com/doc/current/controller.html#the-request-object-as-a-controller-argument

Naudoti `createFromGlobals` yra bloga praktika – pajausi, kai ateis testavimo paskaitos.